apis/exoscale/v1/dbaas_exoscale_mysql.go

```diff
@@ -50,20 +50,20 @@ type ExoscaleMySQLParameters struct {
 	Backup ExoscaleDBaaSBackupSpec `json:"backup,omitempty"`
 }
 
-type ExoscaleMySQLServiceSpec struct {
+type MySQLServiceSpec struct {
 	ExoscaleDBaaSServiceSpec `json:",inline"`
 
 	// +kubebuilder:validation:Enum="8"
 
-	// MajorVersion contains the major version for MySQL.
+	// MajorVersion contains the major version for ExoscaleMySQL.
 	// Currently only "8" is supported. Leave it empty to always get the latest supported version.
 	MajorVersion string `json:"majorVersion,omitempty"`
 
-	// MySQLSettings contains additional MySQL settings.
+	// MySQLSettings contains additional ExoscaleMySQL settings.
```

Comment:
Also here, I'd leave the comments, especially when they show up in the CRD description

Reply:
Fixed